indico/modules/rb_new/schemas.py

```diff
@@ -52,6 +52,20 @@ class Meta:
         fields = ('value', 'title')
 
 
+class AdminRoomAttributesSchema(mm.ModelSchema):
+    attribute_object = {'attribute.name': 'attribute.value'}
```

Comment:
that actually works? nice to know! do you have a link to the marshmallow docs where this is documented?